src/common/SmartUrlInput.js

```diff
@@ -178,8 +151,8 @@ export default function SmartUrlInput(props: Props): Node {
   );
 
   const [prefix, , suffix] = autocompleteRealmPieces(value, {
-    domain: defaultDomain,
-    protocol: defaultProtocol,
+    domain: 'zulipchat.com',
+    protocol: 'https://',
```

Comment:
These get duplicated at this second commit. I believe it's important that they have the same values in those two different places, so both should refer to a common name.

Simplest may be to just have locals with the same names as the old props. When/if a later commit makes one of those locals be referred to in just one spot, it can still be inlined then.